### PR TITLE
Open all sections by default

### PR DESCRIPTION
**Pull request by JiaoJiao123**
Currently, reviewers need to open changed sections one by one, which is trivial. 
In this PR, we open all sections by default upon entering review mode.

### CODE REVIEW

**Comment by JiaoJiao123** [comment]
Issues left: treatment section is not opened

File: web/yo/app/views/gene.html [hunk 1]
```diff
@@ -131,7 +131,7 @@ <h3 ng-if="reviewMode">Gene Type:
   </div>
   <div id="mutation">
     <uib-accordion close-others="false">
-      <div ng-repeat="mutation in gene.mutations.asArray()" ng-init="geneStatus[$index] = {}; geneStatus[$index].isOpen=false; geneStatus[$index].oncogenic = false; geneStatus[$index] = {}" class="accordionMarginTop" ng-hide="status.hideAllObsolete && mutation.name_eStatus.get('obsolete') === 'true'">
+      <div ng-repeat="mutation in gene.mutations.asArray()" ng-init="geneStatus[$index] = {isOpen: false};" class="accordionMarginTop" ng-hide="status.hideAllObsolete && mutation.name_eStatus.get('obsolete') === 'true'">
```
Comment (zhx828) [inline web/yo/app/views/gene.html]: Why removed `geneStatus[$index].oncogenic = false` ?

Comment (JiaoJiao123) [inline web/yo/app/views/gene.html]: because it is initialized in line 192. It is duplication

**Comment by zhx828** [comment]
@JiaoJiao123 before merging this, could you check if we have 50 places have been changed, will this auto-open feature makes the page freeze? Thanks.

**Comment by JiaoJiao123** [comment]
@zhx828 Thank you very much for the reminder! It did make the page freeze when switching back to regular mode from review for big genes. 
And it has been addressed by setting open to false when exiting review.

**Pull request merged**